src/tests/ftest/scripts/main.sh

```diff
@@ -38,6 +38,14 @@
 
 set -eux
 
+# check that vm.max_map_count has been configured/bumped
+if [ $(sudo sysctl -n vm.max_map_count) != 1000000 ] ; then
```

Comment:
Rather than checking that it's == 1000000, could we just check that it's >= ? That would allow for more flexibility